lua/kubectl/views/jobs/definition.lua

```diff
@@ -0,0 +1,99 @@
+local M = {
+  resource = "jobs",
+  display_name = "Jobs",
+  ft = "k8s_jobs",
+  url = { "{{BASE}}/apis/batch/v1/{{NAMESPACE}}jobs?pretty=false" },
+  hints = {
+    { key = "<grr>", desc = "restart", long_desc = "Create job from job" },
+    { key = "<gd>", desc = "desc", long_desc = "Describe selected job" },
+    { key = "<enter>", desc = "pods", long_desc = "Opens pods view" },
+  },
+}
+local hl = require("kubectl.actions.highlight")
+local time = require("kubectl.utils.time")
+
+function M.processRow(rows)
+  local data = {}
+
+  if not rows or not rows.items then
+    return data
+  end
+
+  if rows and rows.items then
+    for _, row in pairs(rows.items) do
+      local job = {
+        namespace = row.metadata.namespace,
+        name = row.metadata.name,
+        completions = M.getCompletions(row),
+        duration = M.getDuration(row),
+        containers = M.getContainerData(row, "name"),
+        images = M.getContainerData(row, "image"),
+        selector = M.getSelector(row),
+        age = time.since(row.metadata.creationTimestamp, true),
+      }
+
+      table.insert(data, job)
+    end
+  end
+  return data
+end
+
+function M.getHeaders()
+  local headers = {
+    "NAMESPACE",
+    "NAME",
+    "COMPLETIONS",
+    "DURATION",
+    "AGE",
+    "CONTAINERS",
+    "IMAGES",
+    "SELECTOR",
+  }
+
+  return headers
+end
+
+function M.getDuration(row)
```

Comment:
In general, these are internal functions inside the module, we shouldn't export them unless needed ofc, not a big deal though

Reply:
Done, made them local and hoisted them up.
f97cd4a..e8a63f0